lib/ansible/module_utils/vmware.py

```diff
@@ -29,20 +30,41 @@ class TaskError(Exception):
     pass
 
 
-def wait_for_task(task):
+def wait_for_task(task, max_backoff=64, timeout=3600):
+    """Function to wait for given task using exponential back-off algorithm.
```

Comment:
Rephrase: `Wait for given task to finish using exponential back-off algorithm.`
Don't use `Function ...`, because it's useless: the reader can see that it's a function just one line above, when writing useful metadata/comments focus on what might be really important to the end-user of this API. This ensures they'll have a pleasant experience when using your module's API.

Reply:
OK.